### PR TITLE
Add various extra resources

### PR DESCRIPTION
**Pull request by jamesbt365**
Adds some extra resources from a suggestion on reddit.

Renames Rewind to its actual name instead of a generic "Replay Analyzer".

### CODE REVIEW

**Comment by Ruben172** [comment]
https://old.reddit.com/r/osugame/comments/1fc4acr/a_curated_collection_of_links_to_various_osu/lm60k9y/ relevant reddit comment

File: content.md [hunk 1]
```diff
@@ -57,6 +58,7 @@ This list is also available online at: https://resources.osucord.moe/
 - [osuck](https://skins.osuck.net/) - Website to get skins for all game modes
 - [Skinship's skin compendium](https://compendium.skinship.xyz/) - Easy to search archive of all osu! skins posted in the Completed Skins forum
 - [osuskins](https://osuskins.net/) - Compendium of osu! skins, especially from top players
+- [OsuSkinMixer](https://github.com/rednir/OsuSkinMixer) - A tool to mix various elements of skins within seconds
```
Comment (Ruben172) [inline content.md:61]:
```suggestion
- [OsuSkinMixer](https://github.com/rednir/OsuSkinMixer) - A tool to easily mix various elements of skins
```

File: content.md [hunk 2]
```diff
@@ -15,8 +15,9 @@ This list is also available online at: https://resources.osucord.moe/
 ### Replay and Score Analysis
 - [Danser-Go](https://github.com/Wieku/danser-go) - Render your replays locally
 - [o!rdr](https://ordr.issou.best/) - Render your replays online (powered by Danser-Go!)
-- [Replay Analyzer](https://github.com/abstrakt8/rewind) - Analyze replays in meticulous detail
+- [Rewind](https://github.com/abstrakt8/rewind) - Analyze replays in meticulous detail
 - [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - Given a replay, displays where the misses happened (also available as a Discord bot)
+- [osu-aim-analyzer](https://github.com/rgbeing/osu-aim-analyzer) - An alternative replay analysis tool to help analyze aim mistakes
```
Comment (Ruben172) [inline content.md:20]:
```suggestion
- [osu-aim-analyzer](https://github.com/rgbeing/osu-aim-analyzer) - A replay analysis tool to help analyze aim mistakes
```

File: content.md [hunk 3]
```diff
@@ -70,13 +72,15 @@ This list is also available online at: https://resources.osucord.moe/
...
+- [ArrowVortex](https://arrowvortex.ddrnl.com/) - Alternative map editor for various VSRGs
 ---
 ### Browser Extensions
 - [osuplus](https://osu.ppy.sh/community/forums/topics/408541?n=1) - Extension that adds extra features to the osu! website
 - [score inspector](https://github.com/darkchii/score-inspector-extension) - A userscript that adds more stats to user profiles on the official osu! website (Extension compatible with the [osu! scores inspector](https://score.kirino.sh/))
 - [osu! preview](https://github.com/JerryZhu99/osu-preview) - Preview a beatmap without downloading it
 - [osu! subdivide nations](https://github.com/Cavitedev/osu-subdivide-nations) - Display the regional flags and names of players on the official osu! website as well as other unofficial sites (Extension compatible with [osu! World](https://osuworld.octo.moe/))
 - [ezpp!](https://github.com/oamaok/ezpp) - Calculate how much PP a map is worth for a particular score
+- [osu-color-changer](https://userstyles.world/style/1767/osu-color-changer) - various colour and stylistic changes for the osu! website, featuring many customization options
```
Comment (Ruben172) [inline content.md:83]:
```suggestion
- [osu-color-changer](https://userstyles.world/style/1767/osu-color-changer) - A userstyle allowing various colour and stylistic changes for the osu! website, featuring many customization options
```

**Review by Ruben172 — CHANGES_REQUESTED** [review]
not me requesting a change on a description I already approved

**Review by Ruben172 — APPROVED** [review]

**Review by oinkrr — APPROVED** [review]

**Pull request merged**